packages/mobile/src/components/details-tile/DetailsTileActionButtons.tsx

```diff
@@ -125,18 +112,18 @@ export const DetailsTileActionButtons = ({
 
   const shareButton = (
     <IconButton
-      fill={neutralLight2}
+      fill={neutralLight4}
```

Comment:
are these supposed to be 4 or 2? was told in one of the syncs that these are all supposed to be 2

Reply:
So I was double checking with Sammie and she said to stick with 4, which is the "diminished" color, and later when we do design system change the diminished color to higher contrast color. So for time being keep with the default color for icons